mysql/errcode.go

```diff
@@ -1036,6 +1036,7 @@ const (
 	ErrWarnMemoryQuotaOverflow             = 8063
 	ErrWarnOptimizerHintParseError         = 8064
 	ErrWarnOptimizerHintInvalidInteger     = 8065
+	ErrUnsupportedSecondArgumentType       = 8066
```

Comment:
Is this error only applicable in `JSON_OBJECTAGG`?

Reply:
Yes